osquery/bro/BrokerManager.cpp

```diff
@@ -0,0 +1,312 @@
+#include <osquery/flags.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/system.h>
+
+#include <broker/broker.hh>
+#include <broker/endpoint.hh>
+#include <broker/message_queue.hh>
+
+#include "osquery/bro/utils.h"
+
+#include "osquery/bro/BrokerManager.h"
+#include "osquery/bro/QueryManager.h"
+#include <iostream>
+#include <list>
+#include <sstream>
+#include <stdlib.h> /* srand, rand */
+#include <time.h>
+
+namespace osquery {
+
+BrokerManager* BrokerManager::_instance = nullptr;
+
+BrokerManager::BrokerManager() {
+  this->qm = QueryManager::getInstance();
+}
+
+osquery::Status BrokerManager::setNodeID(const std::string& uid) {
```

Comment:
You're in the `osquery` namespace so feel free to drop the `osquery::` references.